components/Reviews.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { AllReviews } from "../utils/data";
+
+const Reviews = () => {
+  const [allReviews, setAllReviews] = useState([]);
+    useEffect(() => {
+    AllReviews()
+      .then(({ data }) => {
+          setAllReviews(data);
+          console.log(data)
+      })
+      .catch((err) => console.log(err));
+  }, []);
+    const { reviews } = allReviews;
+  if (reviews) {
+    return (
+      <>
+        <h2> Reviews </h2>
+        <ul>
+          {reviews.map((review) => {
+            return (
+              <li key={review.review_id}>
+                <p>{review.title}</p>
+                <p>{review.designer}</p>
+                <p>{review.owner}</p>
+                <p>{review.review_img_url}</p>
```

Comment:
Currently, your image is just a p tag with text. Do you want to render the image itself using an img tag? :)